src/rdf.test.ts

```diff
@@ -231,41 +233,35 @@ describe("fromRdfJsDataset", () => {
               },
             },
           },
-        },
-        [acrGraphIriString]: {
+        }),
+        [acrGraphIriString]: expect.objectContaining({
           [subject2IriString]: {
             url: subject2IriString,
             type: "Subject",
             predicates: {
               [predicate1IriString]: {
                 blankNodes: [
-                  {
-                    [predicate1IriString]: {
-                      literals: {
-                        [xmlSchemaTypes.string]: [literalStringValue],
-                      },
-                    },
-                  },
-                  {
-                    [predicate1IriString]: {
-                      literals: {
-                        [xmlSchemaTypes.string]: [literalStringValue],
-                        [xmlSchemaTypes.integer]: [literalIntegerValue],
-                      },
-                    },
-                    [predicate2IriString]: {
-                      literals: {
-                        [xmlSchemaTypes.integer]: [literalIntegerValue],
-                      },
-                    },
-                  },
+                  expect.stringMatching(/_:/),
+                  expect.stringMatching(/_:/),
                 ],
               },
             },
           },
-        },
+        }),
       },
     });
+    const subjectsExcludingBlankNodes = getThingAll(
```

Comment:
so am I understanding correct that the representation would have have the blank nodes linked off other legit nodes, if you have `acceptBlankNodes: false` would you still expect those Blank nodes linked or would they just be gone entirely?

Reply:
nvm answered belowed :point_down: